tests/implementor/test_table_run.py

```diff
@@ -37,20 +48,37 @@ class TestRunSuccessfulInit:
     def setup_method(self):
         pass
 
+    @pytest.fixture
+    def token(self):
+        token = Token(name="owner", permission="admin", hashedToken="hash")
+        token.save()
```

Comment:
You can insert a new entry to the database by using `Token.create()` or by initializing it first and saving it with `.save()` later 😇

Reply:
Yes, but does it need to be saved in the database for this test? Just asking, because I did not in my tests and it also worked

Reply:
I thought it could cause an error, when I want to reference a Token from a SimulationConfiguration with foreign key (see the following test case) 

https://github.com/BP2022-AP1/bp2022-ap1/blob/6263c730a63dce9038242b96eb88eba1b9567441/tests/implementor/test_table_run.py#L21)

Reply:
Nevermind, I though about something else, but does not seem to be a problem, so I'm good :)

Reply:
Great 🙌 Thanks for the question, tho 😄